src/widgets/about-mentors/ui/about-mentors.module.scss

```diff
@@ -0,0 +1,40 @@
+.mentors-register-wrapper {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+}
+
+.picture-wrapper {
+  display: flex;
+
+  @include media-tablet {
+    display: none;
+  }
+}
+
+.picture {
+  position: relative;
+  width: 140px;
+  height: 140px;
+
+  @include media-tablet-large {
+    width: 100px;
+    height: 100px;
+  }
+}
+
+.picture:nth-child(3) {
+  left: -140px;
+
+  @include media-tablet-large {
+    left: -100px;
+  }
+}
+
+.picture:nth-child(2) {
+  left: -70px;
+
+  @include media-tablet-large {
+    left: -50px;
+  }
+}
```

Comment:
I must have misunderstood the criteria in the refactoring issue - need to get rid of parent selectors.
Does this imply using only one css class?
![image](https://github.com/user-attachments/assets/8bb39607-bacf-4a21-b748-3bc731c1a9fa)

Reply:
Let's clarify this point with @Quiddlee

Reply:
@SpaNb4 @MikAleinik in the example above i pointed out that the classes selectors should not contain redundant parent selector

```scss
.parent-class { // .parent-class .some-class {} ✅ valid
  .some-class {}
}

// 👇 this makes no sence because when we nesting the classes this will produce the exact same result
& .some-class {} // .parent-class .some-class {} ❌ invalid 

///////////////////////////

&.some-class {} // .parent-class.some-class {} ✅ valid 

&:nth-child() {} // .parent-class.nth-child() {} ✅ valid
```

Reply:
Got it
Thanks for the clarification!